### ISSUE TITLE
H3. describe a linkage between a region and a publication

### ISSUE THREAD

**Issue by congruili** [USGCRP/gcis-ontology#62]
Relate a term to describe a linkage between a region and a publication.  Maybe make it as a subclass of dcterms:Relation.  Currently, we are using http://schema.org/isRelatedTo; please inform whether is a correct use of this term.

**Comment by zednis** [comment]
@justgo129 can you provide a few examples?  

One option that comes to mind is the use of [`dcterms:subject`](http://dublincore.org/documents/dcmi-terms/#terms-subject) and/or [`dcterms:spatial`](http://dublincore.org/documents/dcmi-terms/#terms-spatial).

**Comment by zednis** [comment]
`schema:isRelatedTo` is defined as a relationship between 2 products, so I do not think it is the correct property to use in this scenario.

from http://schema.rdfs.org/all.ttl

```
schema:isRelatedTo a rdf:Property;
    rdfs:label "Is Related to"@en;
    rdfs:comment "A pointer to another, somehow related product (or multiple products)."@en;
    rdfs:domain schema:Product;
    rdfs:range schema:Product;
    rdfs:isDefinedBy <http://schema.org/Product>;
```

**Comment by justgo129** [comment]
@zednis in response to the need for examples, I am finding that this may be an outdated request.  I think we're using dcterms:Relation now.  As it's a class in-lieu of a property, the need for clarification about a substitute term still stands.  e.g. http://data.globalchange.gov/report/ucollaw-ti-2011.thtml

**Comment by congruili** [comment]
I think that [`dcterms:spatial`](http://dublincore.org/documents/dcmi-terms/#terms-spatial) makes more sense according to the definition: spatial characteristics of the resource.

**Comment by justgo129** [comment]
I'd be fine with that since it has range dcterms:Location which itself has a broad definition.

**Comment by justgo129** [comment]
"dcterms:Spatial" it is.  @lic10 or @zednis please make the appropriate change, merge the change, and close this issue (#62).

**Comment by zednis** [comment]
@justgo129  are you sure we are not currently using dcterms:Relation for regions?  

I found this in the RDF for http://data.globalchange.gov/report/nca1.thtml:

```
## Region(s):
   dcterms:Relation <http://data.globalchange.gov/region/northeast-us>;
```

**Comment by zednis** [comment]
I created two pull requests in gcis repository.

USGCRP/gcis/pull/189 updates `dcterms:Relation` to `dcterms:spatial` for publication->region reference

USGCRP/gcis/pull/190 updates `dcterms:Relation` to `dcterms:relation` for region->publication reference.

I see now why we requested an inverse property to `dcterms:spatial`.  No inverse property is defined by dublin core.  We can use `dcterms:relation` as per the open pull request or define a new property in the GCIS namespace.

**Comment by justgo129** [comment]
I would prefer if we define a new property in the gcis namespace to be related to dcterms:relation in some way.  dcterms:relation seems rather vague to me.

**Comment by zednis** [comment]
Ok, how about this?

```
gcis:spatialOf
  a owl:ObjectProperty ;
  subPropertyOf dcterms:relation ;
  rdfs:comment "spatial characteristics for the resource" ;
  owl:inverseOf dcterms:spatial .
```

If there is consensus to add this add it to the ontology and update USGCRP/gcis#190 to use this.

**Comment by justgo129** [comment]
I really like it.  Do you mean "of the resource?"  +1 on making that update to USGCRP/gcis#190

**Comment by zednis** [comment]
@justgo129 No, the definition of `dcterms:spatial` is "Spatial characteristics of the resource."  In that case "of the resource" implies _this_ (the subject in the RDF statement).

I was hoping to have a similarly worded definition for the reverse association.  My first pass may not have been clear enough though.

**Comment by justgo129** [comment]
I think I get it now.  Yep, could we clarify the proposed definition?

**Comment by zednis** [comment]
"resource with these spatial characteristics"?

**Comment by congruili** [comment]
looks good enough

**Comment by justgo129** [comment]
"resource with contents having these spatial characteristics?"
"spatial characteristics of resource content?"

**Comment by zednis** [comment]
To me "resource with" and "resource with contents having" mean basically the same thing...

How about "a resource related to these spatial characteristics"?

**Comment by justgo129** [comment]
Interesting, I perceive it differently.  I see the resource as something physical or electronic and which contains content.  The content, as opposed to the document containing it, would have the spatial characteristics under discussion (i.e. a physical resource itself doesn't have the characteristics; it's the information contained in the resource with the characteristics).  Basically, I see it as the "this is not a pen" issue.

@rewolfe and @CurtTilmes I'll defer to you on this matter.  This is for relating a region (subject) with spatial characteristics such as states contained in that region (object).

**Comment by zednis** [comment]
With dublin core terms the resource and the resource's content seem to be synonymous.  

The definition of `dct:spatial` is "Spatial characteristics of the resource" not "Spatial characteristics of the resource's contents".  This usage of resource as an informational resource is consistent through DCT and terminology I tried to copy in my comment on the inverse property.

**Comment by justgo129** [comment]
Good catch, @zednis.  I wonder why the resource and its content are treated in dublin core as synonymous as it seems rather counter-intuitive to me given various distinctions in the Semantic Web (e.g. a URL versus the page to which it directs are separate objects in various ontologies).
It would be a good discussion topic for another venue.

Back to this particular issue, @zednis are you proposing the following?:
A resource: subject
something="a resource related to these spatial characteristics" Predicate
the spatial characteristics: object?

This would seem more fitting to https://github.com/USGCRP/gcis/pull/189 than https://github.com/USGCRP/gcis/pull/190.

Maybe rephrase to "a resource is related to these spatial characteristics?"  That makes a bit more sense to me and follows some of the examples of definition formatting that already exist in the GCIS Ontology.  Otherwise, +1 for going with "a resource related to these spatial characteristics" as a definition but I'd really like to hear other opinions before proceeding.  @CurtTilmes @bduggan @aulenbac @rewolfe
